src/mmw/apps/core/mail/backends/boto_ses_mailer.py

```diff
@@ -66,18 +65,16 @@ def _send_message(self, email_message):
                 raise
 
     def _log_message(self, email_message, response):
-        statsd.incr('email.message.success')
         logger.debug('Sent email with subject "%s" to %s: %s' % (
             email_message.subject, email_message.to, json.dumps(response)))
 
     def _log_message_failure(self, email_message, trace):
-        statsd.incr('email.message.failure')
         logger.error('Exception raised while sending email '
                      'with subject "%s" to %s: %s' % (
                          email_message.subject, email_message.to, trace))
 
     def _log_quota(self, quota):
-        statsd.gauge('email.quota', quota)
+        pass
```

Comment:
I think that we should make a decision to replace the implementation here, or remove `_log_quota` altogether. Leaning toward the latter, but wanted to surface it for review.